dashboard-app/src/app/pages/hotel-info/views/hotel-info.html

```diff
@@ -150,24 +150,23 @@
                             <label for="checkIn">Check-in</label>
                                 <div class="input-group" moment-picker="hotel.checkIn" format="HH:mm:ss">
                                     <span class="input-group-addon">
-                                        <i class="octicon octicon-clock"></i>
+                                        <i class="fa fa-clock-o" class="fa-lg"></i>
                                     </span>
                                     <input id="checkIn" name="checkIn" class="form-control" placeholder="Select a time" ng-model="hotel.checkIn" ng-model-options="{ updateOn: 'blur' }" required>
-                                    <div ng-show="hotelInfo.$submitted">
-                                        <label class="help-block error-block basic-block" ng-show="hotelInfo.checkIn.$error.required">Please add Check-in time </label>
-                                    </div>
                                 </div>
+                            <div ng-show="hotelInfo.$submitted">
+                                <label class="help-block error-block basic-block" ng-show="hotelInfo.checkIn.$error.required">Please add Check-in time </label>
+                            </div>
                         <div class="form-group" ng-class="{'has-error' : hotelInfo.checkOut.$invalid && hotelInfo.$submitted, 'has-success' : hotelInfo.checkOut.$valid && hotelInfo.$submitted}">
                             <label for="checkOut">Check-out</label>
                                 <div class="input-group" moment-picker="hotel.checkOut" format="HH:mm:ss">
                                     <span class="input-group-addon">
-                                        <i class="octicon octicon-clock"></i>
+                                        <i class="fa fa-clock-o" class="fa-lg"></i>
```

Comment:
Don't need to add twice class attribute. It is enough to define in one. eg. class="fa fa-lg fa-clock-o".